CHANGELOG.md

```diff
@@ -7,7 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
 
+- Update Transifex config and locale directory settings
```

Comment:
"locale directory settings" - is it ok to take this out? Those changes actually happened on master and not in this pull request! Sorry, just it might be confusing later. And is it ok to say what changed in the transifex config? Just something like "Update Transifex config - changed resource name"